### PR TITLE
Release OptaPlanner 8.0.0.Beta1

### PR DESCRIPTION
**Pull request by rsynek**
- removes links to KIE server and Drools
- updates OptaPlanner links to match the released 8.0.0.Beta1
- updates the XSDs
- optawebs stay on 7.45.0.Final so far

@mbiarnes would you please stop updating the optaplanner-website during the 7.x releases from now? The website will point to 8.x artifacts when we merge this PR.

### CODE REVIEW

File: xsd/benchmark/benchmark-8.xsd
Comment (rsynek) [inline xsd/benchmark/benchmark-8.xsd:3]: If you generate the XSD on Java 11 via JAXB, it unfortunately adds unnecessary lines. It does not suffer from this issue on Java 8

File: download/download.html.haml
Comment (ge0ffrey) [inline download/download.html.haml:164]: There's "execution-server-nightly" on line 176/164 a few lines below. I think that can be removed too.

File: learn/documentation.html.haml
Comment (ge0ffrey) [inline learn/documentation.html.haml:42]: There's a "execution-server-nightly" a few lines below too

File: xsd/benchmark/benchmark-8.xsd
Comment (ge0ffrey) [reply xsd/benchmark/benchmark-8.xsd:3]: Do want want to clean up that up manually? or even automatically?
Do those extra lines hurt anyway except for the download size? Few people read the actual XSD.
Proposal: if there's an easy fix in the jaxb configuration, let's do that. If there's not, let's just keep the empty lines, instead of manually (or even automatically) cleaning it up.

**Review by ge0ffrey — APPROVED** [review]
Approved if comments are dealt with. Your call on how you want to handle the XSD empty lines situation.

File: xsd/benchmark/benchmark-8.xsd
Comment (triceo) [reply xsd/benchmark/benchmark-8.xsd:3]: +1 on keeping the lines. We're a Java 11 project now, so whatever 11 generates is the law.

Comment (rsynek) [reply xsd/benchmark/benchmark-8.xsd:3]: There is no easy fix on the JAXB side. This time I removed the lines as I am releasing the XSD manually. Next time, it will be just updated here with the empty lines.

File: download/download.html.haml
Comment (rsynek) [reply download/download.html.haml:164]: good catch, thank you.

**Comment by mbiarnes** [comment]
@rsynek OK - will try to think in this. Otherwise, in case raise a PR for updating
 the website please feel free to close this PR.

**Pull request merged**
